cmd/bcpd/app/app_test.go

```diff
@@ -30,10 +30,10 @@ func TestSendTx(t *testing.T) {
 	chainID := "test-net-22"
 	mainAccount := &account{pk: crypto.GenPrivKeyEd25519()}
 	myApp := newTestApp(t, chainID, []*account{mainAccount})
-
+	var amount int64 = 2000
```

Comment:
:nail_care:  you can use `const` and then you don't have to declare precise type.

Reply:
actually it turned out that I don't need this variable at all, I'd say we need to clean this up anyway, I don't really like how this is currently designed